baby-bear/src/x86_64_avx2.rs

```diff
@@ -0,0 +1,1153 @@
+use core::arch::x86_64::{self, __m256i};
+use core::iter::{Product, Sum};
+use core::mem::transmute;
+use core::ops::{Add, AddAssign, Div, Mul, MulAssign, Neg, Sub, SubAssign};
+
+use p3_field::{AbstractField, Field, PackedField};
+
+use crate::BabyBear;
+
+const WIDTH: usize = 8;
+const P: __m256i = unsafe { transmute::<[u32; WIDTH], _>([0x78000001; WIDTH]) };
+const MU: __m256i = unsafe { transmute::<[u32; WIDTH], _>([0x88000001; WIDTH]) };
```

Comment:
Nit: Maybe add another comment that `MONTY_BITS` is now implicitly 32 and so `MU := P^-1 (mod 2^32) = 0x88000001`.